docs/modules/ROOT/pages/index.adoc

```diff
@@ -7,15 +7,15 @@ TIP: Solace Quarkus Extension for integrating with Solace PubSub+ message broker
 
 == Installation
 
-If you want to use this extension, you need to add the `com.solace.quarkus:quarkus-solace` extension first to your build file.
+If you want to use this extension, you need to add the `com.solace.quarkus:quarkus-solace-client` extension first to your build file.
```

Comment:
We should also add quarkus-solace-messaging-connector reference as well

Reply:
@ozangunalp are we adding this in this PR?

Reply:
I think overall it'd be better to rewrite that doc page. This is just a replace for the artifact-id

Reply:
Okay, probably we will cover this as part of #37